### DISCUSSION TITLE
Github Profile Page Public Email NOT Showing!

### DISCUSSION THREAD

**Discussion by Sang-Buster** [community/community#36483] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

My GitHub profile page does not show my public email when I'm signed out of my account, it only shows it when I'm logged in. When I am logged in, no other devices can see my public email either. I used other devices to check it, still don't see it, so I don't think it's a browser cache issue.

I have configured the `Public Email` to my email. Under `Email Settings`, I've also added my email to my email list, verified it, and set it as my primary email. `Keep my email addresses private` option is also unchecked. Both `git config user.name` and `git config --global user.name` are the same as of my public email. I am clueless about how to solve this issue.

Anyone's help will be appreciated!
Thanks!!!!!

**Comment by jfullstackdev** [comment]
since you want it public, can you have the screenshot of that in your settings? so we can take a look also, 

it should be showing when you set it

for example me, I cannot select the other email and it's not in the dropdown for the Public Email option because it's not verified, I'm just keeping it so no one will take it

**Comment by Sang-Buster** [comment, marked as answer]
Another person on a different discussion answered my question. Apparently, GitHub does not let anyone see your email unless you're signed in. Let's hope in the future they'll give us an email address option for guest viewers😔
